examples/ice_controller/ice_controller.c

```diff
@@ -1679,8 +1691,7 @@ IceControllerResult_t IceController_AddIceServerConfig( IceControllerContext_t *
                                                         IceControllerIceServerConfig_t * pIceServersConfig )
 {
     IceControllerResult_t ret = ICE_CONTROLLER_RESULT_OK;
-    IceControllerResult_t dnsResult;
-    int i;
+    int copyCount = 0;
```

Comment:
Variable copyCount name is not very descriptive? Something like validIceServerCount could make it easier to understand.

Reply:
Done. Thanks for suggestion.